### DISCUSSION TITLE
convert.py ending with "Killed" at lm_head layer

### DISCUSSION THREAD

**Discussion by Christopheraburns** [turboderp/exllamav2#178] [Q&A]
Python 3.10.12 | Ubuntu20.04 | CUDA 11.8 | NVIDIA RTX ADA 4000 (20gb) 
I am converting the new zephyr-7b (beta) with exLlamav2 with the following syntax:

`python exllamav2/convert.py \
    -i zephyr-7b-beta \
    -o quant \
    -c wikitext-test.parquet \
    -b 5.0`

It seems to quantize fine but at the lm_head layer it ends with "Killed".  If I try to load the quantized model with the included test_inference.py I get the error:

`ValueError:  ## Could not find lm_head.* in model`

Which makes sense since given where the process is killed.  How can I troubleshoot this further?  Thanks!

**Comment by fgdfgfthgr-fox** [comment]
Have you checked the data integrity of your model file?
